docs/tutorials/trajectories.rst

```diff
@@ -0,0 +1,67 @@
+==========================
+Handle Expert Trajectories
+==========================
+
+For imitation learning we need expert trajectories.
+In this library they are stored in a :class:`Trajectory <imitation.data.types.Trajectory>` dataclass:
+
+.. code-block:: python
+
+    @dataclasses.dataclass(frozen=True)
+    class Trajectory:
+        obs: np.ndarray
+        """Observations, shape (trajectory_len + 1, ) + observation_shape."""
+
+        acts: np.ndarray
+        """Actions, shape (trajectory_len, ) + action_shape."""
+
+        infos: Optional[np.ndarray]
```

Comment:
Or document it in the code

Reply:
It is called `info` just like the `info` dict returned by the [`step()` method of the environments](https://gymnasium.farama.org/api/env/#gymnasium.Env.step). I totally agree that the name is not very descriptive, but I still think it is a good idea to keep the same name as the gym API dictates.
In fact, the `info` dict is used for all kinds of information that needs to be funneled through the gym API. Therefore it can not really be any more descriptive. It is a questionable part of the gym API ...

I ensured that the shape is defined in the same way as for the fields above, and in code added some more detailed explanation.